modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,124 @@
+:page-categories: Management, Security
+
+Role-based Access Control (RBAC) provides a way to configure permissions for provisioned users at scale. RBAC allows you to manage access for many users to many resources with a streamlined interface. This mechanism works in conjunction with all supported authentication methods.
+
+IMPORTANT: At present, Redpanda only supports RBAC for self-hosted clusters. Configuration is only supported through the command line using RPK.
+
+== RBAC
+
+RBAC is designed to address the critical challenge of access management at scale. It alleviates the process of manually maintaining and verifying a set of raw permissions (ACLs) for a user base that may contain thousands of users. RBAC provides you with a method to onboard new employees easier, audit accesses faster, and adapt more readily to evolutions of usage and compliance needs.
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to implement a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all principals with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as the allowed or denied principal respectively.
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
+
+* `user` - All authorized users have this role by default. There are no ACLs assigned to this role until you add them. This means users have no permissions to interact with any resources you add to your Redpanda cluster until you add relevant permissions to this or another role.
+* `admin` - By default, no users have this role. The default admin role provides full access to all resources in your Redpanda installation.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs can contain allow permissions, deny permissions, or a combination of both. As a result, users may at times have conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+== Managing users and roles
+
+Redpanda allows you to manage your RBAC configurations through both RPK. Examples of each are given here.
```

Comment:
```suggestion
Redpanda allows you to manage your RBAC configurations using rpk. 
```

Reply:
Not sure what the other method you intended to introduce here--Admin API? You never included it so I revised. I know we have Admin API updates for RBAC. If that is what you intended here, please add and sync with @kbatuigas on links.

Reply:
Oh shoot. Originally this had a writeup for RPK or Console, but Console was removed from the beta. I did a quick scrub of alternatives but missed this. There will be Admin API updates, and I've already spoken briefly with @kbatuigas on it, I intend to have a deeper discussion next week with her. As far as I'm aware, Admin API is not ready for beta either, but we can confirm this with @deniscoady and @oleiman .